Chapter3.html

```diff
@@ -0,0 +1,62 @@
+<html>
+ <ol>
```

Comment:
This is all content. You can't put content in there like this. After the <html> tag comes the <head> tag. After that comes the <body> tag. You put content into the body.